Broker/Publisher.php

```diff
@@ -64,8 +65,8 @@ public function publish($messageType, Message $message, array $overridenConfig =
         $messagePublisher->publish($message, $routingKey);
 
         $this->eventDispatcher->dispatch(
-            MessagePublishedEvent::NAME,
-            new MessagePublishedEvent($messageType, $message, $connection, $exchange, $routingKey)
+            new MessagePublishedEvent($messageType, $message, $connection, $exchange, $routingKey),
+            MessagePublishedEvent::NAME
```

Comment:
There should be a conditional here, the proxy only exists since sf 4.3 and I think this bundle supports 3.4 (but doesn't say so in its composer.json?)